apps/expo-go/android/expoview/src/main/java/host/exp/exponent/kernel/Kernel.kt

```diff
@@ -903,6 +904,15 @@ class Kernel : KernelInterface() {
       ExceptionUtils.exceptionToErrorHeader(exception),
       ExceptionUtils.exceptionToCanRetry(exception)
     )
+    if (exception is ManifestException) {
+      kernelScope.launch {
```

Comment:
may need withContext here or inside the suspend function if you need to execute on background thread. the kernelScope by default running on main thread

Reply:
Agree! Fixed.

Reply:
```kotlin
kernelScope.launch {
        withContext(Dispatchers.IO) {
          ExponentPackageLogger.send(
            exception.manifestUrl,
            ExceptionUtils.exceptionToPlainText(exception),
            ExponentPackageLogger.LogLevel.ERROR
          )
        }
      }
```